File: gjs-archetypes/gjs-c-lib-archetype/src/main/resources/archetype-resources/src/main/c/Sample.c
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )
#set( $capitalizedName = $shortName.substring(0,1).toUpperCase() + $shortName.substring(1))
#set( $includeConfig = "${capitalizedName}Config.h")

${symbol_pound}include "${includeConfig}"
${symbol_pound}include "Sample.h"





// Methods and functions implementation here
bool SmpDoSomething () {
    return true;
}

File: gjs-archetypes/gjs-typescript-lib-archetype/src/main/resources/archetype-resources/src/main/python/Sample.py
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )
#set( $capitalizedName = $shortName.substring(0,1).toUpperCase() + $shortName.substring(1))
${symbol_pound}
${symbol_pound} Library ${capitalizedName}  
${symbol_pound}
${symbol_pound} Module Sample.py
${symbol_pound}


 
def sumar(a: float, b: float) -> float:
    """Devuelve la suma de dos números."""
    return a + b
 
 
def es_par(numero: int) -> bool:
    """Devuelve True si el número es par, False si es impar."""
    return numero % 2 == 0
 